testdata/sqllogictests/xlsx.slt

```diff
@@ -57,3 +57,21 @@ select "HEADING" from read_excel('file://${PWD}/testdata/xlsx/multiple_sheets.xl
 # negatives are not allowed for infer_rows
 statement error
 select * from read_excel('file://${PWD}/testdata/xlsx/multiple_sheets.xlsx', sheet_name => 'other', infer_rows => -1);
+
+# create external table
+
+statement ok
+create schema 'xlsx';
+
+statement ok
+set search_path = 'xlsx';
+
+statement ok
+create external table basic_report from excel options( location='./testdata/xlsx/userdata1.xlsx', file_type = 'xlsx');
+
+# implicitly create 
+statement ok
```

Comment:
seems like it'd be ok to test some of the error cases/ or edge cases just to demonstrate that we get errors (safely) when provided with bad inputs.